packages/components/src/Illustration/Spot/_docs/Spot.stickersheet.stories.tsx

```diff
@@ -0,0 +1,781 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import { Heading } from "~components/Heading"
+import { Text } from "~components/Text"
+import { StickerSheetStory } from "~storybook/components/StickerSheet"
+
+import {
+  AccountBasics,
+  Action,
+  AddImage,
+  AddUser,
+  Assertive,
+  BaselineSurvey,
+  BenefitsSurvey,
+  BlankSurvey,
+  CandidateSurvey,
+  Cautionary,
+  CompanyDetails,
+  ChangeReadiness,
+  ChangeSuccess,
+  Coaching,
+  Company,
+  CustomOnboardSurvey,
+  CustomSurvey,
+  CustomUnattributedSurvey,
+  EmergencyResponse,
+  EmployeeData,
+  EndOfProbation,
+  EngagementSurvey,
+  ExecutiveReportSharing,
+  ExitSurvey,
+  EssentialResilience,
+  EssentialProductivity,
+  InfluentialCommunication,
+  FastAction,
+  Feedback,
+  Gdpr,
+  GeneralOnboardSurvey,
+  HealthAndSafety,
+  InclusionSurvey,
+  Individual180,
+  Individual360,
+  Informative,
+  Insights,
+  InternSurvey,
+  LeaderReportSharing,
+  Leadership180,
+  Leadership360,
+  LeadingThroughCrisis,
+  LeadingChange,
+  Manager180,
+  Manager360,
+  ManagerLearning,
+  ManagerReportSharing,
+  MeetingVoices,
+  Negative,
+  NewWaysOfWorking,
+  OneOnOne,
+  PerformanceDiagnostics,
+  PhasedWeek1OnboardSurvey,
+  PhasedWeek5OnboardSurvey,
+  Positive,
+  Productivity,
+  QuickEngagementSurvey,
+  ReadArticle,
+  RemoteManager,
+  RemoteOnboardSurvey,
+  ReOnboarding,
+  ReportSharing,
+  Resilience,
+  ShareReport,
+  SinglePointOnboardSurvey,
+  SpreadsheetTemplate,
+  Strategy,
+  TakeAim,
+  Team1,
+  Team2,
+  TeamEffectiveness1,
+  TeamEffectiveness2,
+  Timezone,
+  Training1,
+  Training2,
+  Training3,
+  ValuesSurvey1,
+  ValuesSurvey2,
+  Video,
+  ViewReports,
+  WellbeingSurvey1,
+  WellbeingSurvey2,
+  WellbeingSurvey3,
+  Workshop,
+  WellbeingSurvey,
+  Response,
+  RemoteWorkQSet,
+  ReturnToWorkplace,
+  PulseSurvey,
+  PaperPen,
+  DataVisualization,
+  Community,
+  BCorp,
+  Resources,
+  Goals,
+  Communications,
+  ScienceBackedTools,
+  Microphone,
+  PowerfulInsights,
+  ActionPlans,
+  Conversations,
+  Process,
+  SkillsDevelopment,
+  ChangeAgents,
+  Alarm,
+  Fire,
+  Fireworks,
+  FullImport,
+  HrisImport,
+  PartialImport,
+  Starburst,
+  Stop,
+  TrafficCone,
+  Trophy,
+  UnderConstruction,
+  ValueAdd,
+  Recommendation,
+  Objective,
+  London,
+  Melbourne,
+  NewYork,
+  SanFrancisco,
+  Behaviour,
+  Learn,
+  Templates,
+  CalendarSync,
+  SpotProps,
+} from "../"
+
+export default {
+  title: "Components/Illustrations/Spot",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+  },
+} satisfies Meta
+
+const engagementSpots = [
+  {
+    Component: BenefitsSurvey,
+    name: "BenefitsSurvey",
```

Comment:
Originally these had more human readable names, but that doesn't help our consumer know what they will actually be exported as. I was thinking we wouldn't need this and could just use the displayName but that's not actually set for these components so I just when with the more verbose approach of remove the spaces and making sure they're in pascal case / align with the component name